locale/fr_FR/author.xml

```diff
@@ -21,22 +21,22 @@
 	<message key="author.submit.startHereTitle">Commencer une nouvelle soumission</message>
 	<message key="author.submit.startHereLink"><![CDATA[<a href="{$submitUrl}" class="action">Cliquer ici</a> pour vous rendre à la première étape du processus de soumission en cinq étapes.]]></message>
 	<message key="author.submit.step1">Étape 1. Commencer la soumission</message>
-	<message key="author.submit.step2">Étape 2. Télécharger la soumission</message>
+	<message key="author.submit.step2">Étape 2. Tranférer la soumission</message>
```

Comment:
Transférer?

Reply:
"Transférer" is a french translation for Upload (better than "Télécharger" which means more "Download")